### DISCUSSION TITLE
about an execption SCAN_FLAT_VECTOR

### DISCUSSION THREAD

**Discussion by xubobbs** [duckdb/duckdb#14528] [Q&A]
hi,
I test my program and found a query assert .
``` 
err:INTERNAL Error: ScanVector called with SCAN_FLAT_VECTOR but result is not a flat vector
This error signals an assertion failure within DuckDB. This usually occurs due to unexpected conditions or errors in the program's logic.
For more information, see https://duckdb.org/docs/dev/internal_errors
```
what does this mean ?

**Comment by Tishj** [comment]
> For more information, see https://duckdb.org/docs/dev/internal_errors

TL;DR we'd appreciate an issue with a reproduction so we can fix it, Internal Exceptions are not something a user should run into

**Comment by xubobbs** [comment]
I only some log and the db file ( > X G size), and restart it work ok ,so I don't know how to reproduct it now.
